### ISSUE TITLE
(ken_rebuild_branch ) No "global_x_ticks" in multi_panel.py

### ISSUE THREAD

**Issue by HosseinSharifii** [Campbell-Muscle-Lab/PyMyoVent#22]
When no ticks is assigned to the x_display in template_jason_file, the script automatically calculate the ticks and assign it to "global_x_ticks" which is not called later on in the code. 
Issue will be fixed if "global_x_ticks" in line 109 renames to "ticks" that is called all over the code.  

Hossein will fix this once ken's_rebuild branch is merged into the master branch.
